using reactivation keys / link chain (#718)
uyuni-project/uyuni-docs: modules/client-configuration/pages/activation-keys.adoc

Record the details of the key for later use.
Unlike typical activation keys, which are not associated with a specific system ID, keys created here do not show up on the menu:Systems[Activation Keys] page.

When you have created a reactivation key, you can use it with a bootstrap script, with Salt, or the [command]``rhnreg_ks`` command line utility.

For more information about bootstrap scripts, see xref:client-configuration:registration-bootstrap.adoc[].

> With Salt, you set a grain on the Salt client in [path]``/etc/salt/minion.d/susemanager.conf``:

Reviewer: ```suggestion
For Salt clients, set a grain in [path]``/etc/salt/minion.d/susemanager.conf``:
```

> With Salt, you set a grain on the Salt client in [path]``/etc/salt/minion.d/susemanager.conf``:

Reviewer: ```suggestion
For Salt clients, set a grain in [path]``/etc/salt/minion.d/susemanager.conf``:
```

----
grains:
    susemanager:
        management_key: <reactivation-key>
----

The [command]``rhnreg_ks`` command is only available for traditional clients.
It re-registers the client and restores its {productname} settings.

> Only on traditional clients you can combine reactivation keys with activation keys to aggregate the settings of multiple keys for a single system profile.

Reviewer: ```suggestion
On traditional clients, you can combine reactivation keys with activation keys to aggregate the settings of multiple keys for a single system profile.
```

Record the details of the key for later use.
Unlike typical activation keys, which are not associated with a specific system ID, keys created here do not show up on the menu:Systems[Activation Keys] page.

> For Salt clients, after you have created a reactivation key, you can use it as the ``management_key`` grain in [path]``/etc/salt/minion.d/susemanager.conf``. For example:

Reviewer: ```suggestion
For Salt clients, after you have created a reactivation key, you can use it as the ``management_key`` grain in [path]``/etc/salt/minion.d/susemanager.conf``. 
For example:
```

Author: by the time, this is already in.

----
grains:
    susemanager:
        management_key: "re-1-daf44db90c0853edbb5db03f2b37986e"
----

> You can now restart the [command]``salt-minion`` to apply the reactivation key.

Reviewer: ```suggestion
Restart the [command]``salt-minion`` process to apply the reactivation key.
```

??
